lib/sagiri.ts

```diff
@@ -225,9 +274,9 @@ export class Sagiri {
     }
   }
 
-  private hasProp <O extends {}>(obj: O, prop: keyof O) {
+  private hasProp<O extends {}>(obj: O, prop: keyof O) {
     return obj && prop in obj;
   }
 }
 
-export default Sagiri;
\ No newline at end of file
+export default Sagiri;
```

Comment:
just put this on the actual class definition
```suggestion
```

Reply:
I tend to have them separate as it makes it a bit clearer to me but /shrug.

Reply:
I rather have it as `export default class Sagiri` since it cleaner, and the fact we made a redundant export anyways

Reply:
Huh, didn't see that.